tests/objects/test_operation.py

```diff
@@ -146,3 +148,78 @@ def test_event_logs(self, loop, op_for_event_logs, operation_agent, file_svc, da
         ]
         event_logs = loop.run_until_complete(op_for_event_logs.event_logs(file_svc, data_svc))
         assert event_logs == want
+
+    def test_writing_event_logs_to_disk(self, loop, op_for_event_logs, operation_agent, file_svc, data_svc):
+        loop.run_until_complete(data_svc.store(operation_agent))
+        start_time = op_for_event_logs.start.strftime('%Y-%m-%d %H:%M:%S')
+        agent_creation_time = operation_agent.created.strftime('%Y-%m-%d %H:%M:%S')
+        want_agent_metadata = dict(
+            paw='testpaw',
+            group='red',
+            architecture='amd64',
+            username='testagent',
+            location='C:\\Users\\Public\\test.exe',
```

Comment:
If you don't want to deal with manually escaping backslashes you can use raw strings in python:

```python
location=r'C:\Users\Python\test.exe'
```

Reply:
sure, I can change that for better readability

Reply:
Oh it was just a tip/suggestion, certainly not a blocking change request :) But I personally like to use them in instances like this.

Reply:
Gotcha, thanks! I already ended up changing it